### DISCUSSION TITLE
请教skynet http client 的使用经验

### DISCUSSION THREAD

**Discussion by jiangshuai377** [cloudwu/skynet#2009] [Q&A]
问题：访问HTTP校验Token 速度很慢 
排除外因：用python 的 asyncio与aiohttp tps很高，侧面验证不是中台接口问题
目前实现是每次校验都调用了  httpc.post 
有更好的实现方式吗

**Comment by sniper00** [comment]
> 问题：访问HTTP校验Token 速度很慢 
> 
> 排除外因：用python 的 asyncio与aiohttp tps很高，侧面验证不是中台接口问题
> 
> 目前实现是每次校验都调用了  httpc.post 
> 
> 有更好的实现方式吗

Https 吗？用nginx http转https正向代理试试

**Reply by jiangshuai377** [reply]
是https，是https的缘故吗？

**Reply by jiangshuai377** [reply]
抱歉 看错了  不是https

**Comment by cloudwu** [comment]
在网络层写 log , 看时间戳，分析是哪个环节的延迟。

**Reply by jiangshuai377** [reply]
嗯，在lua层的 internal.lua 打了日志 ，分析是 connect  和 recebody 耗时占满了， 但是connect  和 recebody 都在挂起等消息回来，需要在C层打日志继续分析吗 ，
